Sources/Run/Helpers/Utils.swift

```diff
@@ -98,3 +99,37 @@ extension String {
         return Formatter.iso8601.date(from: self)   // "Mar 22, 2017, 10:22 AM"
     }
 }
+
+extension Dictionary where Key == String {
```

Comment:
I think this is basically a copy of https://github.com/bsuvorov/ServerSideSwift_extensions/blob/master/Sources/ServerSideSwiftExtensions/Extensions.swift#L86

Could you please try to integrate with this package?

@Niks0707  - also please do a code review on this one.